src/pages/eco-score/index.tsx

```diff
@@ -210,8 +198,8 @@ export default function EcoScore() {
           sx={{ width: 200 }}
         >
           {countryNames.map((country) => (
-            <MenuItem value={country} key={country}>
-              {capitaliseName(country)}
+            <MenuItem value={country.id} key={country.id}>
+              {capitaliseName(country.id)}
```

Comment:
What about using the label directly?
```suggestion
              {country.label}
```